src/main/java/com/xceptance/xlt/engine/dns/XltDnsResolver.java

```diff
@@ -99,6 +117,8 @@ public XltDnsResolver()
         recordAddresses = props.getProperty(PROP_RECORD_ADDRESSES, false);
         shuffleAddresses = props.getProperty(PROP_SHUFFLE_ADDRESSES, false);
         pickOneAddressRandomly = props.getProperty(PROP_PICK_ONE_ADDRESS_RANDOMLY, false);
+        ignoreIPv4Addresses = props.getProperty(PROP_IGNORE_IPV4_ADDRESSES, false);
+        ignoreIPv6Addresses = props.getProperty(PROP_IGNORE_IPV6_ADDRESSES, false);
 
```

Comment:
Do we want to error out in case user has configured to ignore both, ipv4 and ipv6? Just to give some aid in case of misconfiguration - it doesn't make any sense to ignore both.

Reply:
Good idea. Now a warning is printed to the logs. Hope that suffices?

Reply:
That should do it. Thanks!